visallo/visallo
# filter out not addable properties when adding a property

Testing Instructions:
- Make a property as non addable
- In the inspector, make sure the property doesn't show up in the list
- In the search filter, make sure the property does show up in the filter list

Points of Regression:

NO CHANGELOG
